mlflow/models/cli.py

```diff
@@ -137,7 +138,11 @@ def build_docker(model_uri, name, install_mlflow):
 
 def _get_flavor_backend(model_uri, **kwargs):
     with TempDir() as tmp:
-        local_path = _download_artifact_from_uri(posixpath.join(model_uri, "MLmodel"),
+        if ModelsArtifactRepository.is_models_uri(model_uri):
+            underlying_model_uri = ModelsArtifactRepository.get_underlying_uri(model_uri)
+        else:
+            underlying_model_uri = model_uri
```

Comment:
nice.  we should probably try to add this into model registry artifact repo eventually, thx for the update!